packages/rpm/entry.sh

```diff
@@ -42,7 +42,7 @@ for ARCH in ${ARCHS[@]}; do
   sed -i "s/\%global ARCH.*/\%global ARCH ${GOARCH}/" ${SRC_PATH}/kubelet.spec
   # Download sources if not already available
   cd ${SRC_PATH} && spectool -gf kubelet.spec
-  /usr/bin/rpmbuild --target ${RPMARCH} --define "_sourcedir ${SRC_PATH}" -bb ${SRC_PATH}/kubelet.spec
+  /usr/bin/rpmbuild --target ${RPMARCH} --define "_sourcedir ${SRC_PATH}" --define "_smp_build_ncpus 1" --define "__os_install_post %{nil}" -bb ${SRC_PATH}/kubelet.spec
```

Comment:
Why these extra defines? The commit message doesn't clarify.

Reply:
They were the most pragmatic solution to issues encountered with some of the cross-compilation targets.

Disabling `__os_install_post` is to stop the standard macros rpmbuild uses from stripping the binaries.  Stripping is fine on x86_64, but not on ARM targets (since their cross-compilation tools aren't installed):
```
+ /usr/lib/rpm/brp-strip /usr/bin/strip
/usr/bin/strip: Unable to recognise the format of the input file `/root/rpmbuild/BUILDROOT/kubelet-1.19.0-0.%{_arch}/usr/bin/crictl'
/usr/bin/strip: Unable to recognise the format of the input file `/root/rpmbuild/BUILDROOT/kubelet-1.19.0-0.%{_arch}/opt/cni/bin/bridge'
```

If deemed important, I can work in the logic to install the correct cross-compiler binutils packages per arch and override per build target.  The patch gets a little larger at that point.

Setting `_smp_build_ncpus` is to work around a dependency issue.  This macro is defined by `build-constraints-rpm-macros`, which has a conflict with `redhat-rpm-macros`:
```
Error: Transaction test error:
  file /usr/lib/rpm/macros.d/macros.build-constraints from install of build-constraints-rpm-macros-1-2.fc36.noarch conflicts with file from package redhat-rpm-config-222-1.fc36.noarch
```

Not defining `_smp_build_ncpus` results in:
```
+ /usr/lib/rpm/check-rpaths
xargs: invalid number "%{_smp_build_ncpus}" for -P option
Try 'xargs --help' for more information.
error: Bad exit status from /var/tmp/rpm-tmp.jLaICn (%install)
```